packages/gafl-webapp-service/src/pages/contact/contact/route.js

```diff
@@ -34,13 +34,13 @@ export const getData = async request => {
     postHint: getPostHint(permission, mssgs),
     content: getContent(permission, mssgs, junior),
     emailConfirmation: permission.licensee.preferredMethodOfConfirmation === HOW_CONTACTED.email,
-    emailText: getEmailText(permission, mssgs),
+    emailText: getEmailText(permission, mssgs, junior),
     mobileConfirmation: permission.licensee.preferredMethodOfConfirmation === HOW_CONTACTED.text,
-    mobileText: getMobileText(permission, mssgs),
+    mobileText: getMobileText(permission, mssgs, junior),
     licensee: permission.licensee,
     isPhysical: isPhysical(permission),
     errorMessage: getErrorText(permission, mssgs, junior),
-    twelveMonthLicence: permission.licenceLength === '12M'
+    twelveMonthAdultLicence: !junior && permission.licenceLength === '12M'
```

Comment:
Slightly misleading property name, as it includes senior licences too. Use `twelveMonthAdultOrSeniorLicence` or `twelveMonthNonJuniorLicence`